src/image_processing/image_processor.py

```diff
@@ -42,5 +43,6 @@ def _apply_commands(self, image: np.ndarray, command_queue: List[Command]) -> np
         """
         for command in command_queue:
             if command.kernel_type in self._kernel_map:
+                logging.info("Applying command: %s with parameters: %s", command.kernel_type, command.parameters)
```

Comment:
may be good idea to use f-string?

Reply:
Yes, but it is prohibited
https://pylint.readthedocs.io/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html

Reply:
Okay